src/index.js

```diff
@@ -17,10 +17,10 @@ const getDynamicTypes = ({ elementsMap, widgets }) => {
     });
   }
   if (widgets && widgets.length) {
-    widgets.forEach((manifest, index) => {
+    widgets.forEach(({ className, events, members }) => {
```

Comment:
remind me again, why do we pass the manifests to corvid-types instead of passing declaration files generated elsewhere ?

Reply:
IMO widgets should be handled exactly like the elementsMap. it is related to wix biz logic and we will reuse the generation typings logic in the local mode.